doc/Makefile

```diff
@@ -104,16 +101,18 @@ htmlhelp:
 	@echo "Build finished; now you can run HTML Help Workshop with the" \
 	      ".hhp project file in $(BUILDDIR)/htmlhelp."
 
-latex:
-	mkdir -p $(BUILDDIR)/latex $(BUILDDIR)/doctrees
+latex: logo
+	mkdir -p $(BUILDDIR)/latex $(BUILDDIR)/doctrees-latex
```

Comment:
The `-latex` was missing but of course this did not cause harm.

Note that the entire line could/should be removed for this and other similar build targets, because `sphinx-build` does not need it.  Maybe it did in the distant path.